Sources/StreamChat/Query/ChannelListQuery.swift

```diff
@@ -81,7 +81,7 @@ public extension FilterKey where Scope: AnyChannelListFilterScope {
     static var memberCount: FilterKey<Scope, Int> { "member_count" }
     
     /// A filter key for matching the `team` value.
-    static var team: FilterKey<Scope, TeamId> { "team" }
+    static var team: FilterKey<Scope, TeamId?> { "team" }
```

Comment:
Would be useful to get a high-level helper:
```swift
public extension Filter where Scope: AnyChannelListFilterScope {
    ...

    /// Filter to match channels that are not related to any team.
    static var noTeam: Filter<Scope> {
        .equal(.team, to: nil)
    }
}
```
so that when building a query an integrator does not care about details:
```swift
let query = ChannelListQuery(filter: .noTeam)
// vs
let query = ChannelListQuery(filter: .equal(.team, to: nil))
```

Reply:
@loloop This change would be nice since it is more consistent with our other filter `.nonEmpty`. Btw, thank you for your PR 🙏